config/kibana.yml

```diff
@@ -1,8 +1,11 @@
 # Kibana is served by a back end server. This setting specifies the port to use.
 # server.port: 5601
 
-# This setting specifies the IP address of the back end server.
-# server.host: "0.0.0.0"
+# Specifies the host name to which the Kibana server will bind.
+# The default is 'localhost'. With this setting, the Kibana-server will reject any incoming requests from remote users.
```

Comment:
localhost isn't necessarily 127.0.0.1/rejecting remote requests, thoughts on something like "The default is localhost, which usually means other machines won't be able to connect"?